intructions.ts

```diff
@@ -0,0 +1,95 @@
+import { join } from 'path'
+import * as sinkStatic from '@adonisjs/sink'
+import { ApplicationContract } from '@ioc:Adonis/Core/Application'
+
+/**
+ * Prompt choices for the DBMS selection
+ */
+const DBMS_PROMPT_CHOICES = [
+  {
+    name: 'sqlite' as const,
+    message: 'SQLite',
+  },
+  {
+    name: 'mysql' as const,
+    message: 'MySQL / MariaDB',
+  },
+  {
+    name: 'psql' as const,
```

Comment:
Shouldn't we simply call is `pg`. The one that's already in the config file?

Reply:
We can rename it, this string is only used in our script, so it doesn't really matter.

Reply:
Yup, but just for the sake of consistency, it will be nice to have similar names. Also, there are some formatting issues and hence tests are failing.

Reply:
Done!